packages/quantic/force-app/examples/main/lwc/exampleQuanticPager/exampleQuanticPager.html

```diff
@@ -0,0 +1,28 @@
+<template>
+  <c-example-layout
+    title={pageTitle}
+    description={pageDescription}
+    show-preview={isConfigured}>
+  
+    <div slot="configuration">
+      <c-configurator options={options} ontryitnow={handleTryItNow}></c-configurator>
+
+      <div class="slds-var-m-top_large">
+        <lightning-button
```

Comment:
Without Locker Service, we could trigger a search query directly from Cypress by doing something like that:

    cy.window((win) => {
      // we would get the engine from `win.coveoHeadless['the-engine-name']`
      // we would build the controller with `win.CoveoHeadless.buildSearchBox(engine, options)`
      searchBox.updateText('some query');
      searchBox.submit();
    });

But since Locker Service blocks the access to `win.coveoHeadless` and `win.CoveoHeadless`, we have to build the Headless controller from an LWC component. In this case, we use a dedicated button embedded in `exampleQuanticPager` to do so.

We might want to create a specific utility component if we need to do this in more test suites.

Reply:
Could it be added to the configurator?

Reply:
or the exampleLayout?

Reply:
@nathanlb I'm not sure how often we'll want to trigger contextual actions like the "perform search" we have here. Given our typical use cases are related to search, I guess we might need this button when testing other components as well.

I wouldn't add the contextual actions directly in the configurator component, because obviously not all components require the same actions. However, I was thinking of moving "perform search" in a separate component, and adding a slot in the configurator to put these contextual actions. So, one could only add the actions that are relevant to a component.

What do you think?

Reply:
I think that sounds pretty slick! So a like an action toolbar. Considering these tests will act as an example of our preferred way of implementing tests would you add that in now? Or do you prefer doing it separately?

Reply:
Sounds good 👍